cdk_emqx_cluster/cdk_emqx_cluster_stack.py

```diff
@@ -33,6 +33,9 @@
     "eu-west-1": "ami-08edbb0e85d6a0a07",  # ubuntu 20.04 latest
 })
 
+ubuntu_arm_ami = ec2.MachineImage.from_ssm_parameter('/aws/service/canonical/ubuntu/server/focal/stable/current/amd64/hvm/ebs-gp2/ami-id')
```

Comment:
I believe they are switched now: `ubuntu_arm_ami` points to `/amd64/` and `ubuntu_x86_64_ami` points to `/arm64/`